.gitignore

```diff
@@ -9,3 +9,4 @@ node_modules/
 .cache/
 _site/
 tests/_artifacts/
+.DS_Store
```

Comment:
Let's do this with another PR.

Reply:
Why? This is where the problem was raised, this is as good a place as any to add a file to gitignore.
It doesn't warrant a separate PR, since it is not a controversial nor complex change.
Atomicity is good, but there is no need to take it to extremes.

Reply:
That's because your commit message `Added primary changes to add page for Removed icons` will show behind the `.gitignore` for a long time. And it doesn't seem to be related to the `.gitignore` file. It would be better if it shows `Ignore .DS_Store` for `.gitignore`.

<img width="479" alt="CleanShot 2021-12-14 at 11 02 23@2x" src="https://user-images.githubusercontent.com/8186898/145925470-eef52d7e-d156-42ac-a6fd-ce4d91364d9e.png">

Reply:
> That's because your commit message `Added primary changes to add page for Removed icons` will show behind the `.gitignore` for a long time. And it doesn't seem to be related to the `.gitignore` file. It would be better if it shows `Ignore .DS_Store` for `.gitignore`.
> 
> <img alt="CleanShot 2021-12-14 at 11 02 23@2x" width="479" src="https://user-images.githubusercontent.com/8186898/145925470-eef52d7e-d156-42ac-a6fd-ce4d91364d9e.png">

ah, I understand. OK, that makes sense.